paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentSheetTopBarState.kt

```diff
@@ -46,7 +46,8 @@ internal object PaymentSheetTopBarStateFactory {
             StripeR.string.stripe_edit
         }
 
-        val showEditMenu = screen is SelectSavedPaymentMethods && canEdit
+        val showEditMenu =
+            (screen is SelectSavedPaymentMethods || screen is PaymentSheetScreen.ManageSavedPaymentMethods) && canEdit
```

Comment:
in BaseSheetViewModel, canEdit is determined by looking at the PaymentOptionItems. So this is making ManageSavedPaymentMethods indirectly rely on PaymentOptionItems, which we don't really want. Filed a jira to fix: https://jira.corp.stripe.com/browse/MOBILESDK-2168